react-time-track/src/views/project.js

```diff
@@ -34,10 +37,23 @@ const card = {
   width: "auto"
 };
 
-function Project({ project_id }) {
+function Project({ project_id }, props) {
```

Comment:
Can you explain me why you are using that props named props?

Reply:
I forgot to delete this piece of code. Thanks.